partcad-cli/src/partcad_cli/click/command.py

```diff
@@ -75,6 +78,9 @@ def cli(ctx, verbose, quiet, no_ansi, package, format):
     ╚═╝     ╚═╝  ╚═╝╚═╝  ╚═╝   ╚═╝    ╚═════╝╚═╝  ╚═╝╚═════╝
 
     """
+
+    sentry_sdk.start_transaction(op="process", name="PartCAD")
```

Comment:
I need to start transaction outside the function, otherwise decorator does not work properly.
```
[sentry] DEBUG: [Tracing] Starting <process> transaction <PartCAD>
DEBUG:sentry_sdk.errors:[Tracing] Starting <process> transaction <PartCAD>
```